### DISCUSSION TITLE
Donors recognition, criteria, and ways to apply (if applicable)

### DISCUSSION THREAD

**Discussion by adamjonas** [dennisreimann/bitcoin-donations#12] [Ideas]
Bitcoin Dev List is a great start and an asset to the community. Thank you for putting this together. 

In my experience, making the leap to working on open-source bitcoin/LN projects full-time requires a sustainable base of income that usually comes from a grant. It would be useful to allow a way for funders (both whales and companies) to signal they are looking for people/projects to fund and what their criteria might be. Between the [BitMEX blog post](https://blog.bitmex.com/who-funds-bitcoin-development/) and [polylunar](https://polylunar.com/bitcoin-grants-tracker/) there is now better tracking for who is being funded (outside of grassroots) and I think those entities should be celebrated. Additionally, creating a space where contributors can find information on where to apply would be valuable. If applying to exchange A for a grant, there should be a place to monitor what are the deadlines, who to contact, etc. At the very least, it could link out to the source. 

These don't all have the live in the same place, but it'd be somewhat helpful if they did. That makes it easier to plug on podcasts/hackathons/etc. and easier to explain to exchange X that wants to jump in with their first grant or contributor Y who built an awesome wallet and is looking for financial help to keep it going. The current grant process is quite an opaque process to most, and I think Bitcoin Dev List has the potential to serve as a place for the funding matchmaking that is, for the most part, happening behind the scenes.

**Comment by modl21** [comment]
some solid thoughts here, I think maybe we could hit some low hanging fruit to start and then work our way up from there

- link to alternative donation options such as square crypto and the human rights foundation
- link to polylunar existing grant list or provide some type of table on our own
- allow users to sort devs by tags, think: UX, lightning, privacy, multisig, protocol work, etc
- encourage devs to use a donation page that allows multiple options and potentially the ability to subscribe to updates - I really like the btcpay server crowdfund template but ultimately that is up to the individuals on whether or not they use that
- a web of trust type system that allows individuals to vouch for others

**Comment by dennisreimann** [comment]
We'll def need some kinda sorting once this list grows beyond 50 people. It will, and that's great :)

**Comment by ChristopherA** [comment]
Some comments if you add an organization page.

A number of individuals are partially or fully supported by organizations. In my case, I, @Fonta1n3 and others have personal projects that are separate from those of @BlockchainCommons who also fund our work, and @BlockchainCommons also has funded other individual bitcoin contributors in the past.

I note that some individuals have their donation pages belong to a larger organization — Patron's may wish to have some clarity here. 

Some organizations are registered charitable non-profits in certain jurisdictions, and donations by individual may be eligible for charitable deductions for citizens and residents of that jurisdiction. Some non-profits are not registered charitable, but being a non-profit can mean they can receive money from foundations and other non-profits.

Organizations also have more complex donation options. For instance, @BlockchainCommons can accept monthly fiat patronage through GitHub, or single payment of bitcoin via a BTCPay page, and as we are not a registered charity, we have worked with individual donors in the past to run funds through other organziations that can offer a charitable deduction for their nation.

I think it would be of great use to the Patron community to understand this complex landscape better and offer clarity to donors.

-- Christopher Allen

**Comment by polylunar** [comment]
@ChristopherA and @adamjonas are making some great points there and I actually think we could think a bit bigger.
This list is a great thing and already has provided exposure to some devs and gotten them some funding. In the past there have been some initiatives that got some initial traction but then lost steam, i'm hoping this time is different and hence I want to propose to consider a bigger portal-like site with several contributors promoting Bitcoin development and simplifying the funding process.
As @modl21 I'm a fan of low-hanging fruit and hence here some ideas for main elements of such a page:

- similar to the page by @dennisreimann and @modl21 a directory of devs, their main focus, links, donations pages etc
- directory/tracker of bigger funding activities, interlinking to devs, and with as much data as possible
- best practice guides for funders, in the direction of what @ChristopherA was proposing, to make it easier for new funders to understand the process, identify relevant areas of bitcoin development they want to fund, how to identify the right devs; kind of a checklist to work through
- above section can be extended with example contracts (bitmex...), dispute resolution advice, how to channel funds depending on jurisdiction, practical things that make it easier for funders to provide funding
- then i'd think dev profiles would be awesome, to get to know them, like what forbes did with Amiti but less cringe ;) Good stories are needed and we shouldn't be above telling good stories to get more funding in the space, it's about people after all
- then a big one for newbies that are interested in bitcoin development but don't know how to get started. Lots of good information out there but it needs some organizing to make it more accessible, and also different "career" paths in the space

So I think there's lots of good work to be done and we should take time to fledge this out. There's interest in the space on this topic and it's worth getting it right so I'd love to hear everyone's thoughts on the above.

Cheers
poly

**Comment by jansiebert1** [comment]
The nice thing would be if one could quickly recognize from the site whether the developer is predominantly a reviewer or codes new things or is rather a researcher since, for example, reviewing is known to be underfunded. Funds could then flow more to the known bottlenecks.
